tests/option_constants_test.py

```diff
@@ -164,9 +164,16 @@ def test_gssapi_delegation_options(self):
     def test_sslversion_options(self):
         curl = pycurl.Curl()
         curl.setopt(curl.SSLVERSION, curl.SSLVERSION_DEFAULT)
+        curl.setopt(curl.SSLVERSION, curl.SSLVERSION_TLSv1)
+        curl.close()
+
+    # SSLVERSION_SSLv* return CURLE_BAD_FUNCTION_ARGUMENT with curl-7.77.0
+    @util.removed_in_libcurl(7, 77, 0)
+    @util.only_ssl
+    def test_sslversion_options(self):
```

Comment:
Thank you, this method needs to have a different name due to linter complaints.

Reply:
Indeed, should be fixed now.